src/node_process_events.cc

```diff
@@ -21,7 +21,14 @@ using v8::Value;
 Maybe<bool> ProcessEmitWarningSync(Environment* env, std::string_view message) {
   Isolate* isolate = env->isolate();
   Local<Context> context = env->context();
-  Local<String> message_string = OneByteString(isolate, message);
+  Local<String> message_string;
+  if (!String::NewFromUtf8(isolate,
+                              message.data(),
+                              NewStringType::kNormal,
```

Comment:
You'll need to add the `using v8::NewStringType` to the list above for this to work :-)

Reply:
Thank you very much for your support I learned new things 🙏